docs/tutorial.rst

```diff
@@ -190,7 +190,8 @@ BASH_COMPLETIONS   ``[] or ['/etc/...']``      This is a list of strings that
                                                specifies where the BASH
                                                completion files may be found.
                                                The default values are platform
-                                               dependent, but sane.
+                                               dependent, but sane.  To specify an alternate list,
```

Comment:
Sorry, my editor is set to remove trailing spaces, do you want me to revert these other changes?